tests/cli/test_integration.py

```diff
@@ -1,23 +1,53 @@
+import sys
 import pytest
 import os
 import shutil
+import threading
+import ert_shared
 from ert_shared.main import ert_parser
 from argparse import ArgumentParser
 from ert_shared.cli.main import run_cli
-from ert_shared.cli import ENSEMBLE_SMOOTHER_MODE
+from ert_shared.cli import ENSEMBLE_SMOOTHER_MODE, TEST_RUN_MODE
 
+if sys.version_info >= (3, 3):
+    from unittest.mock import Mock
+else:
+    from mock import Mock
 
-def test_target_case_equal_current_case(tmpdir):
-    test_file = os.getenv('PYTEST_CURRENT_TEST')
-    test_folder = os.path.dirname(test_file)
-    data_folder = os.path.join(test_folder, '../../test-data/local')
-    shutil.copytree(os.path.join(data_folder, 'poly_example'), os.path.join(str(tmpdir), 'poly_example'))
+@pytest.fixture()
+def mock_cli_run(monkeypatch):
+    mocked_monitor = Mock()
+    mocked_thread_start = Mock()
+    mocked_thread_join = Mock()
+    monkeypatch.setattr(threading.Thread, "start", mocked_thread_start)
+    monkeypatch.setattr(threading.Thread, "join", mocked_thread_join)
+    monkeypatch.setattr(ert_shared.cli.monitor.Monitor, "monitor", mocked_monitor)
+    yield mocked_monitor, mocked_thread_join, mocked_thread_start
+
+
+@pytest.mark.usefixtures("source_root")
+def test_target_case_equal_current_case(tmpdir, source_root):
```

Comment:
The `@pytest.mark.usefixtures` lines are superfluous; pytest will automatically figure out that you want to use the `source_root` fixture based on test arguments.